src/Markdown.scss

```diff
@@ -1,4 +1,5 @@
-@import 'config/style/overides';
+@use 'variable_overrides';
```

Comment:
Not strictly necessary, but I think we should load the overrides in every top-level file that uses variables, because I'm not sure if it's defined behavior for the Sass compiler to have been smart enough to already load it.